docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/movement/gather-elements-6.rst

```diff
@@ -99,7 +99,7 @@ Example 3 ``indices`` has lesser (than ``data``) shape:
 
 * **1**:  Tensor of type *T*. This is a tensor of a ``rank >= 1``. **Required.**
 * **2**:  Tensor of type *T_IND* with the same rank as the input. All index values are expected to be
-  within bounds ``[0, s-1]``, where ``s`` is size along ``axis`` dimension of the ``data`` tensor. **Required.**
+  within bounds ``[-s, s-1]``, where ``s`` is size along ``axis`` dimension of the ``data`` tensor. **Required.**
```

Comment:
(Optional within this PR, can be discussed/reviewed separately)

As it is an update of behavior within the same op version (GatherElements-6), it would be good to add a note with the release version of the change like:
```
.. note::

    Behavior before 2025.3 OpenVINO release: Negative indices were not supported.
```
Example of similar note for Squeeze-1 #25488, 
https://docs.openvino.ai/2025/documentation/openvino-ir-format/operation-sets/operation-specs/shape/squeeze-1.html